lnbits/server.py

```diff
@@ -1,14 +1,10 @@
-import uvloop
-from uvicorn.supervisors import ChangeReload
-
-uvloop.install()
```

Comment:
- please help me understand, what was this line doing and why it is not required now

Reply:
see commit description: https://github.com/lnbits/lnbits/pull/1860/commits/ec2aed001d79bec044fc46644bc55212f18c0a48